types/snowball-stemmers/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for snowball-stemmers 0.6
+// Project: https://github.com/mazko/jssnowball#readme
+// Definitions by: Ryan Volum <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class Stemmer {
+    stem(term: string): string;
+}
+
+export function newStemmer(locale: string): Stemmer;
```

Comment:
Missing function algorithms as per https://www.npmjs.com/package/snowball-stemmers#example

Reply:
@sheetalkamat I've now included a function for algorithms